packages/docs/site/docs/main/guides/for-theme-developers.md

```diff
@@ -4,4 +4,225 @@ slug: /for-theme-developers
+
+```json
+{
+	"steps": [
+		{
+			"step": "installTheme",
+			"themeZipFile": {
+				"resource": "url",
+				"url": "https://github-proxy.com/proxy/?repo=Automattic/themes&branch=trunk&directory=assembler"
+			},
+			"options": {
+				"activate": true
+			}
+		}
+	]
+}
+```
+
+A blueprint like the one above could be passed to a Playground instance [in several ways](/blueprints/using-blueprints).
+
+## Setting up a demo theme with Blueprints
+
+When providing a link to a WordPress Playground instance with a specific theme activated, you may also want to customize the initial setup for that theme.
+
+By providing a [blueprint](/blueprints/getting-started) to Playground you'll be able to load, activate, and configure a theme. Through [`steps`](/blueprints/steps) in the blueprint, you can configure an initial setup of your theme in the Playground instance.
+
+### `resetData`
+
+With [`resetData`](/blueprints/steps#resetData) step, you can remove the default content of a WordPress installation in order to import your own content.
+
+```json
+"steps": [
+	...,
+	{
+		"step": "resetData"
+	},
+	...
+]
+```
+
+### `writeFile`
+
+With [`writeFile`](/blueprints/steps#resetData) step, you can write data to a file at a specified path. You may want to use this step to write custom PHP code in a PHP file inside the `mu-plugins` of the Playground WordPress instance, so the code is executed automatically when the instance is loaded.
+One of the things you can do through this step is to enable pretty permalinks for your Playground instance:
+
+```json
+"steps": [
+	...,
+	{
+		"step": "writeFile",
+		"path": "/wordpress/wp-content/mu-plugins/rewrite.php",
+		"data": "<?php /* Use pretty permalinks */ add_action( 'after_setup_theme', function() { global $wp_rewrite; $wp_rewrite->set_permalink_structure('/%postname%/'); $wp_rewrite->flush_rules(); } );"
+	},
+	...
+]
+```
+
+### `updateUserMeta`
+
+With [`updateUserMeta`](/blueprints/steps#updateUserMeta) step, you can update any user metadata. For example, you could update the metadata of the default `admin` user of any WordPress installation:
+
+```json
+"steps": [
+	...,
+	{
+		"step": "updateUserMeta",
+		"meta": {
+			"first_name": "John",
+			"last_name": "Doe",
+			"admin_color": "modern"
+		},
+		"userId": 1
+	},
+	...
+]
+```
+
+### `importWxr`
+
+With [`importWxr`](/blueprints/steps#importWxr) step, you can import your own content via a `.xml` file previously [exported from an existing WordPress installation](https://wordpress.org/documentation/article/tools-export-screen/):
+
+```json
+"steps": [
+	...,
+	{
+		"step": "importWxr",
+		"file": {
+			"resource": "url",
+			"url": "https://raw.githubusercontent.com/wordpress-juanmaguitar/blueprints/install-theme-directory-from-repo/blueprints/install-activate-setup-theme-from-gh-repo/blueprint-content.xml"
+		}
+	},
+	...
+]
+```
+
+It is recommended that the exported `.xml` is uploaded to GitHub repository in the same directory the `blueprint.json` is
```

Comment:
A few corrections and slightly better sentence structure:

It is recommended to upload your exported `.xml` file to the same directory as your `blueprint.json` in your GitHub repository.